springdoc-openapi-webmvc-core/src/main/java/org/springdoc/core/SpringDocWebMvcActuatorConfiguration.java

```diff
@@ -1,20 +0,0 @@
-package org.springdoc.core;
-
-import org.springdoc.api.ActuatorProvider;
-import org.springframework.boot.actuate.endpoint.web.servlet.WebMvcEndpointHandlerMapping;
-import org.springframework.boot.autoconfigure.condition.ConditionalOnProperty;
-import org.springframework.context.annotation.Bean;
-import org.springframework.context.annotation.Configuration;
-
-import static org.springdoc.core.Constants.SPRINGDOC_SHOW_ACTUATOR;
-
-@Configuration
-@ConditionalOnProperty(name = SPRINGDOC_SHOW_ACTUATOR)
-public class SpringDocWebMvcActuatorConfiguration {
```

Comment:
Moved to `SpringDocWebMvcConfiguration` as internal class, so that we do not need to duplicate all conditions for checking mvc stack